ci.rb

```diff
@@ -411,7 +416,7 @@ def ldflags
           "#{brew_prefix}/opt/glib/lib/libgio-2.0.a",
           "#{brew_prefix}/opt/glib/lib/libgmodule-2.0.a",
           "#{brew_prefix}/opt/glib/lib/libgobject-2.0.a",
-          "#{brew_prefix}/opt/pixman/lib/libpixman-1.a",
+          "#{brew_prefix}/opt/pixman/lib/libpixman-1.dylib",
```

Comment:
It needs to statically link against all non-system dependencies. On several occasions GitHub as removed dependencies without any notification.

Reply:
well, this was the only library:
1. I found by recursive dir listing AND
2. which successfully compiled finally

Reply:
I'm not following. The static library doesn't exist anymore or it doesn't compile with it?

Reply:
yes, there was no `libpixman-1.a` or similar lib anymore. The only one was the `dylib` one...

Reply:
Then I need to give this some thought. Perhaps we need to build that library ourselves.

Reply:
I think building the library ourselves is the quickest solution.